content/swift/concepts/dictionaries/terms/subscript/subscript.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'Subscript'
+Description: 'Accesses the elements of a collection, list or sequence.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+  - 'Computer Science'
+Tags:
+  - 'Dictionary'
+  - 'Properties'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Subscript`** in Swift is a shortcut to access the elements of a collection, list or sequence. It is a syntactic sugar that allows you to access the elements of a collection using the subscript syntax `[]`. You can define subscripts in your own custom types, and the syntax is similar to computed properties. You can define multiple subscripts with different input parameters and return types.
+
+## Syntax
+
+The syntax for defining a subscript in Swift is as follows:
+
+```swift
+subscript(index: Int) -> Int {
+    // return value
+}
+```
+
+Subscripts can be used to provide a convenient way to access the elements of a collection, list, or sequence. They can also be used to provide a shortcut to access the elements of a collection using the subscript syntax `[]`.
+
+For more information, you can refer to the [Swift documentation on subscripts](https://docs.swift.org/swift-book/LanguageGuide/Subscripts.html).

```

Comment:
```suggestion
```

Reply:
Since we've said this earlier, let us not repeat it here. 
And we do not link to external resources

Reply:
Oh, I'm sorry. My bad. Duly noted. Thanks.